### ISSUE TITLE
Could not create CHEAT_USER_DIR after latest update

### ISSUE THREAD

**Issue by roachsinai** [cheat/cheat#434]
Manjaro install from aur python-cheat. And I already have the cheat user dir: `~/Dropbox/.cheat` which worked before.

```
➜  ~ cat .zshrc | grep CHEAT      
export CHEAT_USER_DIR='~/Dropbox/.cheat'
export CHEAT_EDITOR="vim"
export CHEATCOLORS=true
➜  ~ cheat zsh              
Could not create CHEAT_USER_DIR ( ~/Dropbox/.cheat )
➜  ~ 
```

**Comment by roachsinai** [comment]
```
➜  Dropbox la . | grep cheat
drwxr-xr-x  2 roach roach 4.0K 2月   2 02:46 .cheat
➜  Dropbox pwd               
/home/roach/Dropbox
➜  Dropbox 
```

**Comment by chrisallenlane** [comment]
Closing as resolved by #435.

This is now fixed in `cheat@v2.5.1`.
